### DISCUSSION TITLE
Copilot only suggests one line at a time.

### DISCUSSION THREAD

**Discussion by surjithctly** [community/community#17979] [Copilot Conversations]
As you can see from the below screenshot, copilot is only returning one line suggestion. And in all most cases Next & Prev is disabled too. 

I have tried the shortcut `Option + \` on a mac and it did not helped. How to enable full suggestions?

Version: `v1.25.6056`

![image](https://user-images.githubusercontent.com/1884712/172160960-70a28cb9-c579-4895-92f1-dd8d051b6be7.png)

**Comment by surjithctly** [comment]
Any help?

**Comment by melvincwng** [comment]
@surjithctly, try the shortcut 'Cltrl' + 'Enter'.

Pressing these two keys should make GitHub Copilot show you multiple suggestions (if available).

Please try and let me know if it works, thanks :)

**Reply by surjithctly** [reply]
Tried that, but its also showing same.

**Comment by ben-x9** [comment]
I have exactly the same problem. Ctrl + Enter shows 0 suggestions, pressing Tab then Enter one line at a time gives the function I'm looking for but is hopelessly slow. 

Big shame, would be so much more useful if it just spat out the whole function at once. I'm a few days in to the 60 day trial and honestly not sure it's saving me any time. Perhaps once bugs like this are ironed out will be worth it.

Using TypeScript with VS Code.
